blivet/dbus/blivet.py

```diff
@@ -218,3 +226,27 @@ def Commit(self):
             raise dbus.exceptions.DBusException('%s.%s' % (BUS_NAME, e.__class__.__name__),
                                                 "An error occured while committing the "
                                                 "changes to disk: %s" % str(e))
+
+    @dbus.service.method(dbus_interface=BLIVET_INTERFACE, in_signature='ita{sv}', out_signature='o')
+    def Factory(self, device_type, size, kwargs):
```

Comment:
I still haven't decided if sizes should be passed as ``uint64``s or strings over DBus. Using ``uint64`` is common, but we may as well be the first ones doing "the step to the future".